webpack.config.js

```diff
@@ -1,16 +1,15 @@
-const { join } = require('path');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
 
 module.exports = (env) => ({
     entry: './src/index.js',
     output: {
         filename: 'bundle.js',
-        path: join(__dirname, 'public')
+        path: '/public'
```

Comment:
better to use path module, but not join
```
const path = require('path');

.....
output: {
        publicPath: '/',
        path: path.resolve(__dirname, 'build'),
        filename: 'bundle.js'
    },
```

Reply:
Ok. I will change to use `resolve` instead of `join`.